<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="startMenu">
     <property name="styleSheet">
      <string notr="true">QWidget#startMenu
{
background-color: rgb(48, 48, 48);
}</string>
     </property>
     <widget class="QPushButton" name="sendGameRequest">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>510</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#sendGameRequest
{
background-color: rgb(245, 76, 54);
color: black;
border-radius: 10px;
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton#sendGameRequest::hover
{
background-color:rgb(255, 56, 56);
}
</string>
      </property>
      <property name="text">
       <string>Отправить запрос на игру</string>
      </property>
     </widget>
     <widget class="QPushButton" name="acceptGame">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#acceptGame
{
background-color: rgb(245, 76, 54);
color: black;
border-radius: 10px;
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton#acceptGame::hover
{
background-color:rgb(255, 56, 56);
}</string>
      </property>
      <property name="text">
       <string>Принять приглашение на игру</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcomeText">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#welcomeText
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));
color: black;
font: 18pt &quot;Segoe UI&quot;;
border-radius: 20px;
}
</string>
      </property>
      <property name="text">
       <string>Добро пожаловать в онлайн игру - Камень, ножницы и бумага.
Вы можете отправлять и принимать дуэль на игру с другими пользователями, удачи!</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="waitingResponseText">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#waitingResponseText
{
background-color: rgb(245, 76, 54);
color: black;
border-radius: 10px;
font: 14pt &quot;Segoe UI&quot;;
}</string>
      </property>
      <property name="text">
       <string>Ожидание ответа от игрока.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gameMenu">
     <property name="styleSheet">
      <string notr="true">QWidget#gameMenu
{
background-color: rgb(48, 48, 48);
}</string>
     </property>
     <widget class="QLabel" name="chooseText">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#chooseText
{
background-color: rgb(245, 76, 54);
color: black;
border-radius: 10px;
font: 14pt &quot;Segoe UI&quot;;
}</string>
      </property>
      <property name="text">
       <string>Выбери чем будешь играть:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scissorsButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>141</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Ножницы</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rockButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>141</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Камень</string>
      </property>
     </widget>
     <widget class="QPushButton" name="paperButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>141</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Бумага</string>
      </property>
     </widget>
     <widget class="QPushButton" name="acceptChoose">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>430</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#acceptChoose
{
background-color: rgb(245, 76, 54);
color: black;
border-radius: 10px;
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton#sendGameRequest::hover
{
background-color:rgb(255, 56, 56);
}</string>
      </property>
      <property name="text">
       <string>Подтвердить выбор.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
